.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
This seems to be OS X specific. I suggest you setup a global git ignore file as documented [here](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). So you don't have to ignore such files for every project.